feat: emit on image load event for nuxt-img

hansds opened the pull request:
nuxt-image now emits the onLoad event from the contained image

shadow81627 commented:
Use `$listeners` to allow binding to all native events. https://vuejs.org/v2/guide/components-custom-events.html#Binding-Native-Events-to-Components

hansds commented:
> Use `$listeners` to allow binding to all native events. https://vuejs.org/v2/guide/components-custom-events.html#Binding-Native-Events-to-Components

Thanks for the suggestion! I updated the PR.

shadow81627 commented:
It would be nice to have a test for the on load event. The docs and playground could also be updated to include the onload event.

pi0 commented on src/runtime/components/nuxt-img.vue:
```diff
@@ -1,9 +1,5 @@
-    :src="nSrc"
-    v-bind="nAttrs"
-  >
+  <img :key="nSrc" :src="nSrc" v-bind="nAttrs" v-on="listeners" />
```
Is it maybe possible directly using `$listeners` here without creating a computed + new object? Also would be nice adding same fix to `nuxt-picture` :)

hansds replied:
@pi0 I plan on doing these changes sometime soon (also the docs and tests @shadow81627 mentioned) - keep you posted

hansds commented:
Hey @pi0 and @shadow81627 - I fixed your suggestions. Added docs, playground, nuxt-picture and an e2e test.
For the e2e test I wasn't too sure what the best approach was, I opted for verifying a console message, but maybe there's a better way?

Alphability commented:
@pi0 Do you know when you will release those changes?

madsh93 commented:
@pi0 When will this be released? Docs are live, but `@load` is still not working on latest version.

evangelinepapanicola commented:
`@load` still doesn't seem to work with nuxt-picture in the latest version. According to the docs this should work - might be worth updating them as it is misleading.

baptistebriel commented:
Would be great to have @load event on `<nuxt-picture>` 👍🏻
